packages/medusa/src/services/draft-order.ts

```diff
@@ -230,23 +226,23 @@ class DraftOrderService extends BaseService {
    * @param {object} data - data to create draft order from
    * @return {Promise<DraftOrder>} the created draft order
    */
-  async create(data) {
-    return this.atomicPhase_(async (manager) => {
-      const draftOrderRepo = manager.getCustomRepository(
-        this.draftOrderRepository_
+  async create(data: any): Promise<DraftOrder> {
```

Comment:
here the data seams to be a mix between cart en draftOrder, is that it?

Reply:
Yeah! Draft orders are essentially carts under the hood, which is why the create method takes almost exactly the same property as create in CartService. Though, would still like to have this typed as `CreateDraftOrderInput`. 

You should be able to use the [API payload class](https://github.com/medusajs/medusa/blob/7a1e394b9df9aaf95bcc885685f99ede1dd7aaa9/packages/medusa/src/api/routes/admin/draft-orders/create-draft-order.ts#L145) as inspiration.